docs-src/python/how-to-set-a-dynamic-query-in-python.md

```diff
@@ -0,0 +1,39 @@
+---
+id: how-to-set-a-dynamic-query-in-python
+title: How to set a Dynamic Query
+sidebar_label: Set a Dynamic Query
+description: Use `dynamic=True` on the `@workflow.query` decorator to make a Query dynamic.
+tags:
+- dynamic query
+- python sdk
+- code sample
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/dynamic-ent/dynamic_entities/your_dynamic_query_dacx.py. -->
+
+A Dynamic Query in Temporal is a Query that is invoked dynamically at runtime if no other Query with the same name is registered.
+A Query can be made dynamic by adding `dynamic=True` to the `@query.defn` decorator.
+
+The Query Handler must then accept a single argument of type `Sequence[temporalio.common.RawValue]`.
+The [payload_converter()](https://python.temporal.io/temporalio.workflow.html#payload_converter) function is used to convert a `RawValue` object to the desired type.
+
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-python/blob/dynamic-ent/dynamic_entities/your_dynamic_query_dacx.py">View source code</a>
+
+```python
+# ...
+    @workflow.query(dynamic=True)
+    def dynamic_query(self, name: str, args: Sequence[RawValue]) -> str:
+        name = workflow.payload_converter().from_payload(args[0].payload)
+        return self._greeting
+# ...
+        # Querying with an unregistered Query name and passing a payload.
+        current_greeting = await handle.query("unregistered-query", arg="None")
```

Comment:
```suggestion
        current_greeting = await handle.query("unregistered-query", arg="SomeName")
```

Just because `None` can confuse a Python user that sees it regularly in other ways